utils/SwiftBuildSupport.py

```diff
@@ -12,12 +12,16 @@
 
 from __future__ import print_function
 
-import ConfigParser
 import os
 import pipes
 import subprocess
 import sys
 
+if sys.version_info < (3, 0):
+    import ConfigParser
+else:
+    import configparser as ConfigParser
```

Comment:
Rather than checking the version, I think it's better to try to import the module:

```python
try:
    import ConfigParser
except ImportError:
    import configparser as ConfigParser
```

This is similar to the best practice in Objective-C iOS development, in which it's better to check whether an object `respondsToSelector:`, rather than checking the current iOS version.